test/e2e/member_operator_test.go

```diff
@@ -0,0 +1,115 @@
+package e2e
```

Comment:
Let's maybe rename it to `user_account_ctrl_test.go`? Because we will have other tests for other controllers in that operator too which will be better to keep in dedicated test files.
But I'm not familiar with good practices for operator e2e tests. Will need to check other operators to see how to organize e2e tests better.

Reply:
Yes, I also don't know what would be the right name.  Initially, I have kept `useraccount_test.go` only.  Later I renamed to reflect this is e2e test for member_operator.  I guess, we will have better idea when we add few more e2e test in future.  But for now, I would like to rename it to `useraccount_test.go` for two reasons.  First, we already have `useraccount_controller_test.go` and second this e2e test is `UserAccount` entity centric rather than restricted to contoller only.

Reply:
done [9b1b272](https://github.com/codeready-toolchain/member-operator/pull/26/commits/9b1b272ec26e4f6bfef703ca6a6d861e9818f2bb)